http_parser.c

```diff
@@ -2404,6 +2415,7 @@ http_parser_pause(http_parser *parser, int paused) {
    */
   if (HTTP_PARSER_ERRNO(parser) == HPE_OK ||
       HTTP_PARSER_ERRNO(parser) == HPE_PAUSED) {
+    uint32_t nread = parser->nread;
```

Comment:
This looks unused.

Reply:
It's used in the `SET_ERRNO` macro on the next line. I'll add a comment to document that dependency.